lib/Doctrine/Migrations/Configuration/Configuration/Exception/InvalidConfigurationKey.php

```diff
@@ -2,8 +2,9 @@
 
 declare(strict_types=1);
 
-namespace Doctrine\Migrations\Configuration\Exception;
+namespace Doctrine\Migrations\Configuration\Configuration\Exception;
```

Comment:
Is this on purpose? I find the double `Configuration` in the namespace rather strange.

Reply:
It is intentional. I'm also not very happy, but was done to give come structure.  Currently there are 3 types of configurations: Migration configs, Connection configs and Entity manager configs. So the idea was to keep all of them in the same folder, but in 3 subfolders.
If you have another idea, will be happy to change it.

Reply:
How about `Doctrine\Migrations\Configuration\(EntityManager|Connection|Migration)`? `Configuration\Configuration` seems odd, and doesn't convey why we need the additional subnamespace.